docs/docs/guides/configure-command.md

```diff
@@ -0,0 +1,339 @@
+- Saves with your chosen name and description
+- Exports YAML to `agents/` directory
+
+### 🔄 **Updating Existing Configurations**
+When you use `--load` to modify an existing configuration:
+- **Updates the same configuration** (same ID)
+- **No duplicates created** - modifies the existing saved config
+- Maintains creation date but updates modification timestamp
+- Exports updated YAML to `agents/` directory
+
+```bash
+# Load and modify (updates existing, doesn't create new)
+saiki configure --load my-config-id
+
+# The updated config keeps the same ID but has modified content
+```
+
+### ⚠️ **Important Behavior Changes**
+Previously, loading and modifying a configuration would create a new saved configuration with the same name. **This has been fixed** - now it properly updates the existing configuration instead of creating duplicates.
+
+**Additional UX Improvements:**
+- **No More Double-Save Confusion**: Split save/export decisions for clarity - no more "update and export" confusion
+- **Fixed MCP Server Selection Bug**: Now correctly shows all 13 available servers instead of just current ones
+- **Better Selection Display**: Current servers marked with ✓, shows total available count, improved labeling
+- **Navigation Hints**: Clear guidance about using Ctrl+C to cancel and process flow
+- **Contextual Prompts**: Different messages based on whether creating new or modifying existing configs
+- **Debug Logging**: Added logging to help diagnose selection issues if they occur
+
+## Best Practices
+
+### 🔐 **Security**
+- **Use Environment Variables**: API keys stored as `${OPENAI_API_KEY}` format
+- **Never Commit Keys**: Generated files use environment variable references
+- **Review Generated YAML**: Check configurations before sharing
+
+### 📁 **Organization** 
+- **Use `agents/` Directory**: Default location keeps configurations organized
+- **Descriptive Names**: Choose clear names for saved configurations
+- **Version Control Safe**: Generated YAML files safe to commit (no hardcoded keys)
+
+### 🔄 **Workflow**
+1. **Start with Presets**: Use proven MCP server combinations
+2. **Load and Iterate**: Modify existing configurations as needs evolve (now properly updates!)
+3. **Save Templates**: Keep reusable configurations for different projects
+4. **Export for Deployment**: Generate clean YAML files for production
+
+## Environment Setup
+
+Set required environment variables for your chosen provider:
+
+```bash
+# OpenAI
+export OPENAI_API_KEY="sk-..."
+
+# Anthropic  
+export ANTHROPIC_API_KEY="sk-ant-..."
+
+# Google
+export GOOGLE_API_KEY="AIza..."
+
+# Groq
+export GROQ_API_KEY="gsk_..."
+```
+
+## Integration with Other Commands
+
+Generated configurations work seamlessly with all Saiki commands:
+
+```bash
+# Run with generated configuration
+saiki run --config agents/my-agent.yml
+
+# Start server
+saiki server --config agents/my-agent.yml
+
+# Validate configuration
+saiki validate --config agents/my-agent.yml
+```
+
```

Comment:
we moved to `--agent` earlier.

we don't have a `run` sub-command

we also don't have a `validate` sub-command. but it could be a good idea to add this actually